backend/tests/requests/test_census.py

```diff
@@ -0,0 +1,44 @@
+"""Test census information requests for the Encompass API."""
+from backend.app.requests import census
+from backend.lib.database.postgres import connect
+
+import flask
+
+from flask_testing import LiveServerTestCase
+
+import mock
+
+import pytest
+
+engine = connect.create_db_engine()
+
+
+class TestCensusByServiceAreaRequest(LiveServerTestCase):
+    """Test class for census info requests."""
+
+    def create_app(self):
+        """Start a new flask app for testing."""
+        app = flask.Flask(__name__)
+        app.config['TESTING'] = True
+        return app
+
+    @mock.patch('backend.lib.fetch.census.fetch_census_info_by_service_area')
+    def test_census_info_by_service_area_request(self, mock_fetch):
+        """Test census by service area request in a simple case."""
+        service_area_ids = ['ca_alameda_94601']
+        mock_fetch.return_value = {'ca_alameda_94601': {'age': {'0-18 Years': 10.0}}}
+
+        def _mock_get_json(force=True):
```

Comment:
Maybe define this as a tool? mock_get_json that takes a json as input? I think we do this in C2Q